src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clone = { ...state };
+  const change = [];
```

Comment:
The variable `change` could be renamed to `stateHistory` or `stateChanges` to better reflect its purpose of storing the history of state changes.